release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -467,7 +467,32 @@ In the table, features are marked with the following statuses:
 === Removed features
 
 [id="ocp-4-6-bug-fixes"]
-== Bug fixes
+== Bug fixes 
+
+*Web console (Developer perspective)*
+
+* Previously, when you tried to delete a Knative application through the *Topology* view, a false positive error about a non-existing *Knative route* was reported. This issue is now fixed and the error is no longer displayed. link:https://bugzilla.redhat.com/show_bug.cgi?id=1866214[BZ#1866214])
+
+* Previously, Developer Console did not allow images from insecure registries to be imported. This bug fix adds a checkbox that allows user to use the insecure registries in the *Deploy image* form (link:https://bugzilla.redhat.com/show_bug.cgi?id=1826740[BZ#1826740])
+
+* When a user selected the *From Catalog* option to to create an application, the *Developer Catalog* displayed a blank page instead of a list of templates to create an application. This was caused when the 1.18.0 Jaeger operator was installed. This issue has now been fixed and the templates are displayed as expected. link:https://bugzilla.redhat.com/show_bug.cgi?id=1845279[BZ#1845279])
+
+* When adding or deleting a task in a Pipeline through the *Pipeline Builder* in the Developer Console, the interface was rearranging the tasks incorrectly. With this fix the parallel tasks are reconnected with the connector on addition or deletion of a task and the positions of the tasks are not rearranged. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1856155[BZ#1856155])
```

Comment:
Maybe:
```
With this fix, the parallel tasks are reconnected with the connector when
adding or deleting a task; the positions of the tasks are not rearranged.
```

Reply:
Positions of the tasks will be rearranged but the position of the chain (connections) should not be rearranged. 
on **deletion** of  a parallel task, which has some sequential task attached to it, **should be rearranged** in manner where those sequential task is still connected to the pipeline rather than getting disconnected from it.

Reply:
```suggestion
* When deleting a parallel task in a Pipeline through the *Pipeline Builder* in the Developer Console, the interface was rearranging the tasks connected to the parallel task incorrectly, creating orphan tasks. With this fix, the tasks connected to the deleted parallel task are reconnected with the original Pipeline. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1856155[BZ#1856155])
```